jetty-alpn/jetty-alpn-conscrypt-server/src/test/java/org/eclipse/jetty/alpn/conscrypt/server/ConscryptHTTP2ServerTest.java

```diff
@@ -79,11 +79,9 @@
     private void configureSslContextFactory(SslContextFactory sslContextFactory)
     {
         Path path = Paths.get("src", "test", "resources");
-        File keys = path.resolve("keystore").toFile();
+        File keys = path.resolve("keystore.p12").toFile();
         sslContextFactory.setKeyStorePath(keys.getAbsolutePath());
-        sslContextFactory.setKeyManagerPassword("OBF:1vny1zlo1x8e1vnw1vn61x8g1zlu1vn4");
-        sslContextFactory.setTrustStorePath(keys.getAbsolutePath());
-        sslContextFactory.setTrustStorePassword("OBF:1vny1zlo1x8e1vnw1vn61x8g1zlu1vn4");
+        sslContextFactory.setKeyStorePassword("OBF:1vny1zlo1x8e1vnw1vn61x8g1zlu1vn4");
```

Comment:
shall we not use obfuscation at all?

Reply:
No, it's just a test.

Reply:
but we use the password directly sometimes and obfuscate others.   We should just do one or the other everywhere.... but not really important.

Reply:
I updated the tests to never use obfuscated passwords, which are now only used in specific tests and in configuration files (e.g. `realm.properties`, `*.mod` files, etc.).